src/main/java/com/mide/gangsaeng/board/BoardController.java

```diff
@@ -21,4 +24,10 @@ public ResponseEntity<String> write(@RequestBody BoardRequest request) {
         boardService.write(request);
         return ResponseEntity.ok("ok");
```

Comment:
This is actually more related to phase1, but I think there is a better reponse code when we succeed creating board. Could you check and fix?

Reply:
I fixed it to return 201 status code.